cli/internal/lockfile/berry_lockfile.go

```diff
@@ -216,7 +216,7 @@ func (l *BerryLockfile) Subgraph(workspacePackages []turbopath.AnchoredSystemPat
 	}
 
 	// berry only includes a cache key in the lockfile if there are entries with a checksum
-	cacheKey := -1
+	cacheKey := "-1"
```

Comment:
Now that we're using a string for `cacheKey` lets use `""` as the empty value.
```suggestion
	cacheKey := ""
```